### DISCUSSION TITLE
NACA 4412 Validation Case - Mesh File

### DISCUSSION THREAD

**Discussion by OguzhanKaraaslan** [su2code/SU2#1358] [Q&A] [answered]
Hi, I have been trying to validate NASA's NACA4412 Validation case. In the configuration file path under TestCases/deformation/naca4412/def_NACA4412.cfg, I couldn't find the relevant mesh file in this repository, any idea?
ps: i have my own mesh also, however i have some error issues in the results, that's why I need that mesh

**Comment by OguzhanKaraaslan** [comment, marked as answer]
I figured it out the problem, there are two TestCases directory under "su2code" repository, one of them contains the cfg file whereas the other one contains the mesh file. Thanks.

**Reply by TobiKattmann** [reply]
Yep we keep the large files (like meshes and restart files) in a separate repository to keep the size of the code repo at an acceptable level. https://github.com/su2code/Testcases

**Reply by rejirejish** [reply]
Where can I get the configuration file for the TestCases?

**Reply by TobiKattmann** [reply]
The configuration files are in the regular SU2 code repo. Also under Testcases/ 

The folder structures between the two instances are the same so when you located a cfg, the mesh will be under the same relative path in the separate Testcases repo

**Reply by rejirejish** [reply]
Got it thank you very much.
